internal/config.go

```diff
@@ -64,15 +65,42 @@ func LoadConfig(path string) (*Config, error) {
 	return &c, nil
 }
 
-func (c *Config) fillAndValidate() error {
-	if c.Results.Common == nil {
-		if c.Results.FerretDB == nil || c.Results.MongoDB == nil {
-			return fmt.Errorf("both FerretDB and MongoDB results must be set (if common results are not set)")
+// Merge common config with both databases
+func mergeTestConfigs(common, mongodb, ferretdb *TestsConfig) error {
+	if common != nil {
+		ferretdb.Skip = append(ferretdb.Skip, common.Skip...)
+		mongodb.Skip = append(mongodb.Skip, common.Skip...)
+
+		ferretdb.Fail = append(ferretdb.Fail, common.Fail...)
+		mongodb.Fail = append(mongodb.Fail, common.Fail...)
+
+		ferretdb.Pass = append(ferretdb.Pass, common.Pass...)
+		mongodb.Pass = append(mongodb.Pass, common.Pass...)
+
+		if common.Default != "" {
+			if ferretdb.Default != "" || mongodb.Default != "" {
+				return errors.New("default value cannot be set in common, when it's set in database")
+			}
+			ferretdb.Default = common.Default
+			mongodb.Default = common.Default
```

Comment:
remove cyclomatic complexity, do `return`.

Reply:
I don't quite understand. What if `Default` and `Stats` are both set in common?

Reply:
> I don't quite understand. What if `Default` and `Stats` are both set in common?

Let's have a talk.